docs/docs/deploy/heroku.md

```diff
@@ -0,0 +1,170 @@
+---
+title: Deploy to Heroku
+summary: A walkthrough of deploying to heroku
+authors:
+    - Stephen van Beek
+date: 2020-05-27
+---
+
+# Deploying to Heroku
+
+[Heroku](https://www.heroku.com) is a PaaS product that makes deploying your application easy. With it's generous free tier it's a perfect way to kick off a new project.
+
+Springboard works pretty well with Heroku and benefits a fair bit from a number of Heroku add-ons:
+
+* [mailgun](https://mailgun.com) as an SMTP server to send emails;
+* [sentry](https://sentry.io/welcome/) for error-tracking;
+* [logdna](https://logdna.com/) for log aggregation; and 
+* [hosted postgresql](https://www.heroku.com/postgres) as a data store.
+
+You can check the other add-ons heroku comes with out of the box [here](https://elements.heroku.com/addons).
+
+## Minimum Viable Deployment
+
+There are a couple of ways that we can deploy to Heroku - they maintain excellent docs on this [here](https://devcenter.heroku.com/categories/deployment). 
+The deployment mechanism we'll be focusing on is the github integration. We'll be building out a production-ready pipeline here. 
+We'll connect our app to a springboard based github repo and deploy the application with a database.
+
+One of the primary benefits to working in Springboard is simplicity of building/deploying. It's a single, stateless JVM 
+instance that's connected to a database. Most of the tricky first-time setup has already been done in the gradle build scripts. 
+As such, anything that can deploy a spring boot project can deploy a springboard project without difficulty. 
+
+As it happens, Heroku has terrific integration with Spring Boot so deploying a live, production-ready application is pretty simple.   
+
+I'm going to presume you've signed up to a Heroku account, if you haven't [then do so](https://signup.heroku.com/), login to the Heroku dashboard and get ready to go. 
+
+### Step 1: Create a Heroku App
+
+If you've just signed up to Heroku you'll be presented with an empty dashboard. On this page, click the `New` button and select `App`.
+
+[![heroku empty dashboard](/assets/images/heroku-deploy/heroku-empty-dashboard.png)](/assets/images/heroku-deploy/heroku-empty-dashboard.png)
```

Comment:
Other types of tutorials you've done have had "alt" tags with descriptions- is there any need to do that here?

Reply:
The structure of this tag is as follows:

[![alt tag for image](image url)](link to image on click)

So you see! We had alt tags all along...